packages/cursorless-vscode/src/keyboard/TokenTypes.ts

```diff
@@ -52,7 +49,7 @@ export interface TokenTypeValueMap {
   pairedDelimiter: SurroundingPairName;
 
   // action config section
-  simpleAction: SimpleKeyboardActionType;
+  simpleAction: KeyboardActionDescriptor;
```

Comment:
note how we lost some type specificity here; before we knew it was a simple action